src/Compilers/CSharp/Test/Semantic/Semantics/GenericConstraintsTests.cs

```diff
@@ -2447,9 +2447,9 @@ public class B : A
 {
     public override void M<T>() where T : unmanaged { }
```

Comment:
>where T : unmanaged [](start = 32, length = 19)

nit: consider adding a test with multiple constraints (`override ... where T : unmanaged, I`), so each will be reported as an error.

Reply:
> nit: consider adding a test with multiple constraints (override ... where T : unmanaged, I), so each will be reported as an error.

Error_WhenOverride_HasUnmanagedConstraint2 and we report just one error as I would expect.

---
In reply to: [276462660](https://github.com/dotnet/roslyn/pull/35048#discussion_r276462660) [](ancestors = 276462660)

Reply:
if not already covered, maybe also test with other combinations: `override ... where T : class, class`, `override ... where T : class, struct`

---
In reply to: [276462660](https://github.com/dotnet/roslyn/pull/35048#discussion_r276462660) [](ancestors = 276462660)